core/http-auth-aws-crt/src/main/java/software/amazon/awssdk/http/auth/aws/crt/AwsCrtS3V4aHttpSigner.java

```diff
@@ -25,14 +26,14 @@
  * specifically for S3-CRT.
  */
 @SdkPublicApi
-public interface AwsCrtS3V4aHttpSigner extends HttpSigner<AwsCredentialsIdentity> {
+public interface AwsCrtS3V4aHttpSigner extends AwsV4aHttpSigner {
 
     /**
      * Get a default implementation of a {@link AwsCrtS3V4aHttpSigner}
      *
      * @return AwsCrtS3V4aHttpSigner
      */
-    static AwsCrtS3V4aHttpSigner create() {
+    static AwsV4aHttpSigner create() {
```

Comment:
Same question here, why change it to parent type?

Reply:
This one should go away in a follow up PR, but to answer your question `DefaultAwsCrtS3V4aHttpSigner` implements `AwsV4aHttpSigner` and not `AwsCrtS3V4aHttpSigner`.

Reply:
Gotcha, fine with leaving it as is if this is going to be removed anyway.